CITATION.cff

```diff
@@ -0,0 +1,20 @@
+cff-version: 1.2.0
+message: "Hackathon Cookiecutter-based Template"
+authors:
+  - family-names: Grover
+    given-names: Maxwell
+    orcid: https://orcid.org/0000-0002-0370-8974
+    website: https://github.com/mgrover1
+    affiliation: Argonne National Laboratory
+  - family-names: Biddle
+    given-names: Mathew
+    orcid: https://orcid.org/0000-0003-4897-1669
+    website: https://github.com/MathewBiddle
+    affiliation: NOAA Integrated Ocean Observing System Office
+  - family-names: Johnson
+    given-names: Kelli
+    orcid: https://orcid.org/0000-0002-5149-451X
+    website: https://github.com/kellijohnson-NOAA
+    affiliation: NOAA Northwest Fisheries Science Center
```

Comment:
I just changed jobs on Monday so I am now
NOAA Office of Science and Technology

Reply:
Congrats on the new gig! I can update accordingly! Thank you for working with us on this project!!